i7-10810U_chatreey/dev-github/reinstall_playwright.sh

```
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+GIT_ROOT=$(git rev-parse --show-toplevel)
+cd "$GIT_ROOT"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)